### ISSUE TITLE
Maternal inflammatory bowel disease

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7439]
**Preferred term label:**
Maternal inflammatory bowel disease

**Synonyms**

**Definition (free text, please give PubMed ID)**
Occurrence of inflammatory bowel disease in a mother during pregnancy (PMID: 27085078).

**Parent term (use hpo.jax.org/app)**
Prenatal maternal abnormality HP:0002686

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Comment**
Patients with maternal inflammatory bowel disease are known to be at increased risk of adverse neonatal outcomes including increased incidence of preterm birth, low birth weight, congenital anomalies and  neonatal intensive care unit admissions in infants (PMID: 32469057 - A Systematic Review and Meta-analysis).

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by jagcsun** [comment]
Two new child terms may be created for Maternal inflammatory bowel disease:
--Maternal Crohn’s disease
--Maternal ulcerative colitis

PMID: 27933203; 34046084

**Comment by pnrobinson** [comment]
The HPO should not be used to code the past medical history or family history in detail. For individual cases, the Phenopacket could be used for this. The problem is that if we started to do this, we would eventually have one maternal X for every X in MONDO, and the same for paternal, and brother and sister ... etc !
